src/Lumper.Lib/BSP/IO/LumpWriter.cs

```diff
@@ -122,42 +121,20 @@ private LumpHeaderInfo WriteCompressed(Lump lump)
         lzmaStream.Dispose();
 
         mem.Seek(0, SeekOrigin.Begin);
+        long compressedLength = mem.Length;
 
-        if (uncompressedLength == 0 || mem.Length > uncompressedLength)
-        {
-            // No point compressing empty lumps, also LZMA header takes up 22
-            // bytes so useless for very small lumps.
-            if (uncompressedLength > 0)
-            {
-                Logger.Debug(
-                    "Compressed lump larger than uncompressed, skipping. "
-                        + $"(compressed: {mem.Length}, uncompressed: {uncompressedLength})"
-                );
-            }
+        var writer = new BinaryWriter(mem);
+        const int lzmaId = ('A' << 24) | ('M' << 16) | ('Z' << 8) | 'L';
```

Comment:
A better way to write this:
`var lzmaId = Encoding.ASCII.GetBytes("LZMA");`

Reply:
Good shout, though just realized a UTF8 literal works here i.e. `var lzmaID = "LZMA"u8;`, which is a `ReadonlySpan<byte>` which a `BinaryWriter` will treat the actual same way as a `byte[]`.

Updating a few other usages of `Encoding.ASCII` to use `u8` literals.

Reply:
I wasn't aware that was an option, neat.